packages/oak/lib/core/App/index.js

```diff
@@ -86,7 +89,7 @@ export default forwardRef((options, ref) => {
     redo,
     getText,
     getOverrides,
-  }), Object.values(state));
+  }), [...Object.values(state), oakRef]);
```

Comment:
`oakRef` will never change as the ref will update `oakRef.current` not the variable itself. If it works as is, you can remove `oakRef` and revert this line to the previous value